osquery/utils/aws_util.cpp

```diff
@@ -477,4 +492,13 @@ Status appendLogTypeToJson(const std::string& log_type, std::string& log) {
   }
   return Status(0, "OK");
 }
+
+void setAWSProxy(Aws::Client::ClientConfiguration& config) {
+  config.proxyScheme =
```

Comment:
I would prefer this to fail as opposed to revert to the default.

Reply:
@fmanco aside from practically reimplementing the [`SchemaMapper::FromString`](https://github.com/aws/aws-sdk-cpp/blob/da682606e2b693d444c1ed87b40ac23ff3fb7d7a/aws-cpp-sdk-core/source/http/Scheme.cpp#L43-L60) function in the aws-sdk-cpp (which does not return an error, etc), I'm not sure how you'd want this to fail.. any suggestions?

Reply:
Well, if `SchemaMapper::FromString` just takes anything lets just go with it.